pages/ksphere/kommander/1.2/tutorials/configure-rbac/index.md

```diff
@@ -27,84 +27,204 @@ In this procedure, using these default roles, you will create four different per
 
+The two cluster roles for this persona are:
+
+- _View Role_: a `ClusterRole` applies to all target clusters in all workspaces.
+- _Global Kommander View Role_: a `ClusterRole` applies to the management cluster.
+
+#### Using the Kommander UI
+
 In the Kommander UI, do the following:
 
 1. Select _Global_ in the header drop-down.
 2. Select _Administration_ > _Access Control_.
-3. Select the _Cluster Roles_ tab.
-   There are several default roles created by Kommander. The two that will be used for this persona are:
-
-- _View Role_: because this is a Cluster Role type role, it applies to all target clusters in all workspaces.
-- _Global Kommander View Role_: because this is a Kommander Role type role, it applies to the management cluster.
-  Select the Cluster Policies tab and add these two roles to the _Global Viewer_ group.
+3. Select the _Cluster Roles_ tab. Kommander creates several default roles, the ones to select for this persona are _View Role_ and _Global Kommander View Role_.
+4. Select the Cluster Policies tab and add these two roles to the _Global Viewer_ group.
 
 You have now created role bindings that associate the Global Viewer group with these two roles, which ensure that the users in the Global Viewer group have read-only access to the management cluster and all target clusters.
 
+#### Using kubectl
+
+Assuming `kubectl` is attached to your Kommander cluster and you have a `VirtualGroup` named `global-viewer`. You can apply these two `VirtualGroupKommanderClusterRoleBinding` to link the group to roles:
+
+```yaml
+apiVersion: workspaces.kommander.mesosphere.io/v1alpha1
+kind: VirtualGroupKommanderClusterRoleBinding
+metadata:
+  name: workspace-admin-cluster-role-binding
+spec:
+  clusterRoleRef:
+    name: view
+  virtualGroupRef:
+    name: workspace-admin
+---
+apiVersion: workspaces.kommander.mesosphere.io/v1alpha1
+kind: VirtualGroupKommanderClusterRoleBinding
+metadata:
+  name: workspace-admin-kommander-role-binding
+spec:
+  clusterRoleRef:
+    name: kommander-global-view
+  virtualGroupRef:
+    name: workspace-admin
+```
+
 ### Workspace Admin persona
 
 The _Workspace Admin_ persona should have admin access to the Default Workspace. This allows users to administrate all namespaced resources in the workspace namespace on the management cluster, as well as all cluster resources on the workspace’s target clusters.
 
+The two roles for this persona are:
+
+- _Workspace Admin Role_: a `WorkspaceRole` that applies to all target clusters in all workspaces
+- _Kommander Workspace Admin Role_: a `KommanderWorkspaceRole` that applies to the management cluster. This role grants admin access to the workspace namespace on the management cluster. When a `VirtualGroupKommanderWorkspaceRoleBinding` with this role gets created, a `ClusterRole` and `VirtualGroupKommanderClusterRoleBinding` will automatically be created at the global level to allow access to this workspace specifically, instead of all workspaces.
```

Comment:
```suggestion
- _Kommander Workspace Admin Role_: a `KommanderWorkspaceRole` that applies to the management cluster. This role grants admin access to the workspace namespace on the management cluster. When a `VirtualGroupKommanderWorkspaceRoleBinding` with this role gets created, a `ClusterRole` and `VirtualGroupKommanderClusterRoleBinding` automatically create at the global level to allow access to this specific workspace, instead of all workspaces.
```